src/System.Memory/src/System/Buffers/IMemoryList.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#pragma warning disable 1591
```

Comment:
@terrajobst mentioned that doc comments are not required but they seem to be in this project.

Reply:
Yes, we need to add xml commenting for the public surface area. I don't think the warning should be disabled.